src/alloc/mod.rs

```diff
@@ -9,10 +10,12 @@ use core::{
     num::NonZeroUsize,
     ptr::{self, NonNull},
 };
-pub use liballoc::alloc::{handle_alloc_error, Layout};
+use liballoc::alloc::Layout;
```

Comment:
Let's reexport everything, which has already been stabilized.
```suggestion
pub use liballoc::alloc::Layout;
```

Reply:
Oh, sorry! Meant to just not `pub use` `handle_alloc_error`.